style2.css

```diff
@@ -378,7 +415,49 @@ img {
   font-size: 10px; /* Smaller text size */
   opacity: 0.5;
 }
-/* ----------------------------- */
+/* -------------Side Panel---------------- */
+
+.wrapper {
+  display: flex;
+  height: 100vh;
+}
+
+.side-panel-toggle {
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  display: grid;
+  place-items: center;
+
+  color: #ffffff;
+  background: #009578;
+  border: none;
+  border-radius: 5px 0 0 5px;
+  outline: none;
+  cursor: pointer;
+
+  position: absolute;
+  top: 160px;
+  left: 28%;
+}
+
+.side-panel-toggle:hover {
+  background: #007960;
+}
+
+.sp-icon-close {
+  display: none !important;
```

Comment:
Usually it's better not to use the important clause. Instead, try to make your classes more specific (for example, one level more specific), to they get prioritised over other styles.